ci_framework/hooks/templates/service_config_ceph.yaml.j2

```diff
@@ -27,8 +27,36 @@ patches:
     - op: add
       path: /spec/cinder/template/cinderBackup/replicas
       value: {{ cifmw_services_cinder_bkp_replicas | default(1) }}
+    - op: add
+      path: /spec/manila/enabled
+      value: {{ cifmw_services_manila_enabled | default('false') }}
+    - op: add
+      path: /spec/extraMounts/0/extraVol/0/propagation/-
+      value: ManilaShare
     - op: replace
       path: /spec/manila/template/manilaShares/share1/replicas
       value: {{ cifmw_services_manila_shr_replicas | default(1) }}
+    - op: replace
```

Comment:
@viroel can we add to manilaAPI the following:

```
customServiceConfig: |
   [DEFAULT]
   debug = true
   enabled_share_protocols=nfs,cephfs,cifs
```

It should be an extra entry in kustomize:

```
  - op: replace
      path: /spec/manila/template/manilaAPI/customServiceConfig:
      value:  |
                [DEFAULT]
                debug = true
                enabled_share_protocols=nfs,cephfs,cifs
      
```
I see failure in the APIs for quotas and I'm wondering if there's a problem in the `enabled_share_protocols` section of that component.